tests/conftest.py

```diff
@@ -22,11 +22,15 @@ def private_key_to_checksum_address(key):
     ).to_checksum_address()
 
 
-class GenericEthHelper(EthereumGeneric):
+class EthHelperMixin:
```

Comment:
Is this an abstract mixin?!

Reply:
no just a mixin. needed because of EthereumGeneric is now derived from BaseBlock